sky/packages/sky/lib/widgets/basic.dart

```diff
@@ -268,17 +268,23 @@ class Baseline extends OneChildRenderObjectWrapper {
 }
 
 class Viewport extends OneChildRenderObjectWrapper {
-  Viewport({ Key key, this.offset: 0.0, Widget child })
-    : super(key: key, child: child);
+  Viewport({
+    Key key,
+    this.scrollOffset: Offset.zero,
+    this.scrollDirection: ViewportScrollDirection.vertical,
+    Widget child
+  }) : super(key: key, child: child);
 
-  final double offset;
+  final Offset scrollOffset;
```

Comment:
The distinction between Offset scrollOffset and double scrollOffset is confusing.  Can we (eventually?) use the Offset version everywhere?